node/src/accountant/receivable_dao.rs

```diff
@@ -88,269 +121,170 @@ impl ReceivableDao for ReceivableDaoReal {
         ReceivableDaoReal {
             conn,
+            big_int_db_processor: BigIntDbProcessor::default(),
             logger: Logger::new("ReceivableDaoReal"),
         }
     }
 
-    // Question: Why would we not update last_received_timestamp here? Is this a bug?
-    // Answer: No, it's not a bug. Adding more debt is different from receiving a payment.
-    fn try_add_debt(&self, wallet: &Wallet, amount: i64) -> Result<bool, String> {
-        let mut stmt = self
-            .conn
-            .prepare("update receivable set balance = balance + ? where wallet_address = ?")
-            .expect("Internal error");
-        let params: &[&dyn ToSql] = &[&amount, &wallet];
-        match stmt.execute(params) {
-            Ok(0) => Ok(false),
-            Ok(_) => Ok(true),
-            Err(e) => Err(format!("{}", e)),
-        }
-    }
-
-    // Question: We didn't just receive a payment; why are we setting last_received_timestamp?
-    // Answer: All new debts should start out young so that they don't trigger immediate delinquency.
-    // Except for exotic tests, timestamp should be now or in the very recent past.
-    fn try_insert(
-        &self,
-        timestamp: SystemTime,
-        wallet: &Wallet,
-        amount: i64,
-    ) -> Result<(), String> {
-        let mut stmt = self.conn.prepare("insert into receivable (wallet_address, balance, last_received_timestamp) values (?, ?, ?)").expect("Internal error");
-        let params: &[&dyn ToSql] = &[&wallet, &amount, &to_time_t(timestamp)];
-        match stmt.execute(params) {
-            Ok(_) => Ok(()),
-            Err(e) => Err(format!("{}", e)),
-        }
-    }
-
     fn try_multi_insert_payment(
         &mut self,
         timestamp: SystemTime,
         payments: &[BlockchainTransaction],
     ) -> Result<(), ReceivableDaoError> {
         let xactn = self.conn.transaction()?;
         {
-            let mut stmt = xactn.prepare("update receivable set balance = balance - ?, last_received_timestamp = ? where wallet_address = ?")
-                .expect ("Internal SQL error");
             for transaction in payments {
-                let timestamp = dao_utils::to_time_t(timestamp);
-                let gwei_amount = match unsigned_to_signed(transaction.gwei_amount) {
-                    Ok(amount) => amount,
-                    Err(e) => return Err(ReceivableDaoError::SignConversion(e)),
-                };
-                let params: &[&dyn ToSql] = &[&gwei_amount, &timestamp, &transaction.from];
-                stmt.execute(params)
-                    .map_err(|e| ReceivableDaoError::RusqliteError(e.to_string()))?;
+                self.big_int_db_processor.execute(Either::Right(&xactn), BigIntSqlConfig::new(
+                    "update receivable set balance_high_b = balance_high_b + :balance_high_b, balance_low_b = balance_low_b + :balance_low_b, last_received_timestamp = :last_received where wallet_address = :wallet",
```

Comment:
Can you put a comment in here explaining that the values you're adding are always going to be negative? Otherwise this is confusing.